department-of-veterans-affairs/va.gov-cms
# VACMS-3147: Updates Drush Site Status commands for Drush 10.

## Description

See #3147.

I'll test this on other environments when the PR's fully open, but these commands are pretty trivial.

This commit doesn't change the `composer.json` or `composer.lock`, just adds the new files.  I'm going to check in the next little bit to see if these added files interfere with Drush 8.

I looked and didn't see any code or scripts using `va-gov-get-deploy-mode` programmatically, but I might've missed some, and the terseness of the response (`ENABLED` or `DISABLED`) made me think that this might be used somewhere in a script, so I simply `echo`ed that string to output rather than rephrasing it in a friendlier way.

## Testing done
Ran the commands, saw expected output.  Since these are simple passthrough commands, I don't see much value in automated tests.

## Screenshots
<img width="855" alt="Screen Shot 2021-03-03 at 10 07 29 AM" src="https://user-images.githubusercontent.com/1318579/109827303-987fe300-7c09-11eb-9c13-3a7bd629164d.png">
<img width="734" alt="Screen Shot 2021-03-03 at 10 16 20 AM" src="https://user-images.githubusercontent.com/1318579/109827248-90c03e80-7c09-11eb-892c-c3b83b3f8dea.png">

## QA steps
As a developer running Drush 10:
1. Check `lando drush list` and verify that:
    - [ ] the commands appear in the list and have appropriate descriptions.
2. Run the commands and verify that the results are as expected:
    - [ ] `va-gov-get-deploy-mode` should return either `DISABLED` or `ENABLED` as appropriate, as the Drush 8 command did
    - [ ] `va-gov-enable-deploy-mode` should enable deploy mode
    - [ ] `va-gov-disable-deploy-mode` should disable deploy mode

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
